BlazarBase/src/main/java/com/hubspot/blazar/base/BuildConfig.java

```diff
@@ -9,6 +9,7 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.Objects;
 import com.google.common.base.Optional;
+import com.hubspot.mesos.Resources;
```

Comment:
You may introduce a blazar class to hold the required resources for a build. In this class we can also add some checks to make sure we don't allow users to exceed some limits in requested memory and cpu.